meshtastic/admin.proto

```diff
@@ -402,6 +402,11 @@ message AdminMessage {
      */
     SharedContact add_contact = 66;
 
+    /*
+     * Initiate or respond to a key verification request
+     */
```

Comment:
Probably does not need admin in the name as it is an admin message

Reply:
Probably going to have to stay, otherwise I get this when trying to regen
``meshtastic/admin.proto:501:9: "meshtastic.KeyVerification" is already defined in file "meshtastic/mesh.proto".``

Reply:
I suppose I can do ``KeyVerificationAdmin key_verification = 67;``